parts/linux/cloud-init/artifacts/components.json

```diff
@@ -9,6 +9,18 @@
           "latestVersion": "1.8.22",
           "previousLatestVersion": "1.8.20"
         }
+      ],
+      "windowsVersions": []
```

Comment:
Does this need to be here if it's empty? If it's the components.cue will complain, you can consider configure `windowsVersions` as optional too.

Reply:
I don't think it has to be there - I prob put it there out of habit. Will remove.

Reply:
Done.